sdk/src/signer.rs

```diff
@@ -77,7 +77,8 @@ use async_trait::async_trait;
 /// This trait exists to allow the signature mechanism to be extended.
 ///
 /// Use this when the implementation is asynchronous.
-#[async_trait]
+#[cfg_attr(target_arch = "wasm32", async_trait(?Send))]
```

Comment:
Is there any reason not to always define async_trait this way?

Reply:
I think it makes sense if there's a chance that that trait will be used in a Wasm context.